From: PupJune2 <PupJune2@users.noreply.github.com>
Date: Sat, 17 Oct 2026 11:30:00 +0000
Subject: CromIKO More Names

Remove remaining Cromite names from the user interface:
- the ad block filter list maintained by the Cromite project is shown
  as "Experimental filters" (its download address is unchanged)

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 components/adblock/core/subscription/subscription_config.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/adblock/core/subscription/subscription_config.cc b/components/adblock/core/subscription/subscription_config.cc
--- a/components/adblock/core/subscription/subscription_config.cc
+++ b/components/adblock/core/subscription/subscription_config.cc
@@ -288,7 +288,7 @@ const std::vector<KnownSubscriptionInfo>& config::GetKnownSubscriptions() {
        SubscriptionFirstRunBehavior::SubscribeAtFirstRun,
        SubscriptionPrivilegedFilterStatus::AllowedAndChecked},
       {GURL("https://raw.githubusercontent.com/uazo/cromite/master/tools/filters/experimental-cromite-filters.txt"),
-       "Cromite experimental filters",
+       "Experimental filters",
        {},
        SubscriptionUiVisibility::Visible,
        SubscriptionFirstRunBehavior::SubscribeAtFirstRun,
--
